drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -160,16 +173,16 @@ def items_without_answers():
             "url_name": getattr(self, 'url_name', ''),
             "display_zone_labels": self.data.get('displayLabels', False),
             "display_zone_borders": self.data.get('displayBorders', False),
-            "items": items_without_answers(),
-            "title": self.display_name,
+            "items": items_without_answers(self),
+            "title": self._(self.display_name),
```

Comment:
We can remove/change everything in this PR -- really I was more interested in demonstrating the i18n workflow itself, so whatever makes sense to tag/translate is fine with me.  We have a list of approximately two dozen XBlocks that we're going to need to configure for I18N, and I'm sure each one is going to have its own pecularities.